modules/hcp-ec2-client/main.tf

```diff
@@ -25,30 +25,41 @@ resource "aws_security_group_rule" "allow_ssh_inbound" {
   security_group_id = var.security_group_id
 }
 
+resource "aws_security_group_rule" "allow_http_inbound" {
```

Comment:
This allows a user to use their browser to connect to the demo apps without SSH forwarding, and lets us more easily test it as well.

Reply:
What exactly is the demo app? And is it serving https? Ideally it would but totally understand if that's out of scope.